src/components/crossDevice/ClientSuccess/style.scss

```diff
@@ -4,9 +4,9 @@
   @extend %icon-circle-background;
   height: 144 * $unit;
   width: 144 * $unit;
+  margin: 50 * $unit auto;
```

Comment:
Can you add a comment so we can make sure we don't introduce the issue again by mistake